calour/_testing.py

```diff
@@ -54,8 +54,13 @@ def setUp(self):
         self.gnps_cluster_info = join(test_data_dir, 'gnps_clusterinfosummarygroup.txt')
         # a fasta file for testing the AmpliconExperiment
         self.seqs1_fasta = join(test_data_dir, 'seqs1.fasta')
-        # a qiime2 biom table artifact
+        # a qiime2 non-hashed biom table artifact
```

Comment:
do we really need a new data set? how about using one of the existing?

Reply:
we need a new one since we need taxonomies and rep-seqs to also test the incorporation into the loaded experiment

Reply:
we can create the qza files for the existing data set